### PR TITLE
[Admin][Customer] Rework ui of customer's show page

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Before:
zrzut ekranu 2017-02-28 o 12 27 23

[EDIT]
After:
zrzut ekranu 2017-03-01 o 09 26 01

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Customer/show.html.twig [hunk 1]
```diff
@@ -15,5 +15,9 @@
 
     <div class="ui divider"></div>
     {% include '@SyliusAdmin/Customer/Show/_breadcrumb.html.twig' %}
+
+    {#{% include '@SyliusAdmin/Customer/Show/_menu.html.twig' %}#}
```
Comment (NeverResponse) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Customer/show.html.twig]: ?

Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Customer/show.html.twig]: WIP

**Comment by pjedrzejewski** [comment]
If store has a lot of channels, the stats can be a bit overwhelming, maybe we can use tabs with these stats?

**Comment by GSadee** [comment]
@pjedrzejewski changed 👍

**Comment by pjedrzejewski** [comment]
Thanks Grzesiu! 👍

**Pull request merged**